src/main/java/io/github/queerbric/queerbrics/mixin/ItemMixin.java

```diff
@@ -0,0 +1,43 @@
+package io.github.queerbric.queerbrics.mixin;
+
+import java.util.Random;
+
```

Comment:
Rest of the codebase doesn't put empty line inbetween imports

Reply:
This is because no different groups are used in any other file.